packages/blade/src/components/Icons/HomeIcon/HomeIcon.tsx

```diff
@@ -0,0 +1,20 @@
+import { Svg, Path } from '../_Svg';
```

Comment:
If there is an issue with Home icon lets not add it here? Otherwise this PR will remain blocked